strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceTest.java

```diff
@@ -31,6 +31,7 @@
 import org.springframework.util.CollectionUtils;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.carlspring.strongbox.services.support.ArtifactEntrySearchCriteria.Builder.anArtifactEntrySearchCriteria;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
We decided to switch on AssertJ in tests.
Please change assertions to AssertJ way.

Reply:
Done.